packages/frontend/editor-ui/src/components/canvas/elements/nodes/render-types/CanvasNodeAIPrompt.vue

```diff
@@ -120,26 +236,138 @@ async function onSubmit() {
 		background: transparent;
 		resize: none;
 		font-family: var(--font-family);
+		padding: 0;
+
+		/* Custom scrollbar styles */
+		@supports not (selector(::-webkit-scrollbar)) {
+			scrollbar-width: thin;
+		}
+		@supports selector(::-webkit-scrollbar) {
+			&::-webkit-scrollbar {
+				width: var(--spacing-2xs);
+			}
+			&::-webkit-scrollbar-thumb {
+				border-radius: var(--spacing-xs);
+				background: var(--color-foreground-dark);
+				border: var(--spacing-5xs) solid white;
+			}
+		}
 	}
 }
 
-.form_footer {
+.formFooter {
 	display: flex;
 	justify-content: flex-end;
-	padding: var(--spacing-2xs);
 }
 
-.or {
+/* Suggestion Pills Section */
+.pillsContainer {
 	display: flex;
 	justify-content: center;
+	align-items: flex-start;
+	gap: var(--spacing-2xs);
+	flex-wrap: wrap;
+}
+
+.suggestionPill {
+	display: flex;
+	height: 33px;
+	padding: 0 13px;
```

Comment:
We could just use padding to define the height of this container.
```suggestion
	padding: var(--spacing-2xs) var(--spacing-xs);
```